fileuploader-kiwiirc-plugin/plugin.js

```diff
@@ -66,7 +67,21 @@ kiwi.plugin('fileuploader', function (kiwi, log) {
 
     const tokenManager = new TokenManager()
 
+    function loadLocale(_lang) {
+        try {
+            let lang = (_lang ? _lang : kiwi.i18n.language).split('-')
+            lang = lang[0] + '_' + lang[1].toUpperCase()
+            return require(/* webpackMode: "eager" */ '@uppy/locales/lib/' + lang)
+        } catch (e) {
+            return require(/* webpackMode: "eager" */ '@uppy/locales/lib/en_US')
```

Comment:
This should probably also use the `import()` syntax, since we're already using `import` statements instead of `require` everywhere else.